Gems/PhysX/Code/Source/System/PhysXSystem.cpp

```diff
@@ -51,6 +100,30 @@ namespace PhysX
         AZ::AllocatorInstance<PhysXAllocator>::Create(allocatorDescriptor);
 
         InitializePhysXSdk(cookingParams);
+
+        InitializePerformanceCollector();
+    }
+
+    void PhysXSystem::InitializePerformanceCollector()
+    {
+        auto performanceMetrics = AZStd::to_array<AZStd::string_view>({
+            PerformanceSpecPhysXSimulationTime,
+        });
```

Comment:
uber nit/standards: Brace placement (not a big deal and you can ignore just couldn't help mentioning it 😅)

Reply:
That's actually how VS auto-formats it for me (using clang-format?). 
Maybe the global settings need updating to comply with coding standard?

Reply:
Ah that's fine, not sure if we can maybe update it but this isn't a huge problem, thanks for checking